coderd/inboxnotifications.go

```diff
@@ -28,16 +29,63 @@ const (
 	notificationFormatPlaintext = "plaintext"
 )
 
+const (
+	FallbackIconWorkspace = "DEFAULT_WORKSPACE_ICON"
+	FallbackIconAccount   = "DEFAULT_ACCOUNT_ICON"
+	FallbackIconTemplate  = "DEFAULT_TEMPLATE_ICON"
+	FallbackIconOther     = "DEFAULT_OTHER_ICON"
+)
+
+var fallbackIcons = map[uuid.UUID]string{
```

Comment:
alternative data structure:

`fallbackIcons = map[NotificationIcon][]uuid.UUID`

not a must, just a preference